apps/ocsp.c

```diff
@@ -753,7 +757,7 @@ static int add_ocsp_cert(OCSP_REQUEST **req, X509 *cert,
                          STACK_OF(OCSP_CERTID) *ids)
 {
     OCSP_CERTID *id;
-    if (!issuer) {
+    if (issuer == NULL) {
```

Comment:
Since you edited this line, please insert a blank line after the local variable declaration, too,